bindings/profilers/wall.cc

```diff
@@ -1046,10 +1207,15 @@ void WallProfiler::OnGCStart(v8::Isolate* isolate) {
   auto curCount = gcCount.load(std::memory_order_relaxed);
   std::atomic_signal_fence(std::memory_order_acquire);
   if (curCount == 0) {
-    gcAsyncId = GetAsyncIdNoGC(isolate);
+    if (collectAsyncId_) {
+      gcAsyncId = GetAsyncIdNoGC(isolate);
```

Comment:
I understand this is not new, though could you comment on why we are collecting async IDs and why they are stable

Reply:
makes sense, we want to collect the GC events and associate them with a given workload